exampleConfig.js

```diff
@@ -12,5 +12,6 @@ exports.config = {
     verifyCert: true, // OPTIONAL (boolean), should we attempt to verify the server certificate before allowing communication, default true
     timeout: 10000, // OPTIONAL (integer), number of milliseconds to wait for establishing a connection to Instrumental before giving up, default 10s
     recordCounterRates: true, // OPTIONAL (boolean) whether or not to send ".rate" metrics with counters, default true
```

Comment:
I agree with Joel, but at minimum, I think our comment should say our **recommended** setting is false even though the default is true